### PR TITLE
doc: generate v1.13.1

### PR DESCRIPTION
**Pull request by marcosuma**
## Description

Changelog for v1.13.1

Link to any related issue(s): https://jira.mongodb.org/browse/INTMDB-1314

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+- fix!: handles paused clusters with errors when updating. [\#1640](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1640) ([marcosuma](https://github.com/marcosuma))
+- fix: adds `data_process_region` field to `mongodbatlas_online_archive` resource [\#1634](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1634) ([marcosuma](https://github.com/marcosuma))
+
+**Internal Improvements**
+
+- fix: Update issues.yml to remove assignee [\#1649](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1649) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- fix: \(INTMDB-1312\) It is not possible to add breaking change label to PR [\#1647](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1647) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- doc: addresses outstanding comments from 1.13.0 docs release [\#1648](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1648) ([oarbusi](https://github.com/oarbusi))
+- chore: migrates `alert_configuration` to new SDK [\#1630](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1630) ([oarbusi](https://github.com/oarbusi))
```
Comment (Zuhairahmed) [inline CHANGELOG.md]:
```suggestion
- chore: migrates `mongodbatlas_alert_configuration` to new SDK [\#1630](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1630) ([oarbusi](https://github.com/oarbusi))
```

**Review by Zuhairahmed — APPROVED** [review]
few minor commits and suggestion below. otherwise LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+**Enhancements**
+
+- feat: Unit test resource and data source schemas in [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework-benefits) [\#1646](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1646) ([lantoli](https://github.com/lantoli))
+
+**Bug Fixes**
+
+- fix: uses SchemaConfigModeAttr for list attributes in `mongodbatlas_cluster` resource. [\#1654](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1654) ([marcosuma](https://github.com/marcosuma))
+- fix: handles attributes as computed in `mongodbatlas_cluster` resource. [\#1642](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1642) ([marcosuma](https://github.com/marcosuma))
+- fix: avoids error when removing project api key assignment for deleted project [\#1641](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1641) ([AgustinBettati](https://github.com/AgustinBettati))
+- fix!: handles paused clusters with errors when updating. [\#1640](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1640) ([marcosuma](https://github.com/marcosuma))
```
Comment (andreaangiolillo) [inline CHANGELOG.md]: Do we want to list this under breaking change?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**